app/assets/stylesheets/core/_link.scss

```diff
@@ -10,7 +10,7 @@
   }
 
   &:focus {
-    color: $govuk-text-colour;
+    color: $govuk-focus-text-colour;
```

Comment:
I'm assuming this is a slightly different red from the error colour?

Reply:
Nope, this is actually a shade of black in order to generate the focus state consistently regardless of the type of link.

Reply:
Oops - I see now!